integrated_channels/integrated_channel/management/commands/__init__.py

```diff
@@ -58,48 +58,65 @@ def add_arguments(self, parser):
             choices=INTEGRATED_CHANNEL_CHOICES.keys(),
         )
 
-    def get_integrated_channels(self, options, **filter_kwargs):
+    @staticmethod
+    def _get_enterprise_customer(uuid):
         """
-        Generates a list of active integrated channels, filtered from the given options.
-
-        Raises errors when invalid options are encountered.
+        Returns the enterprise customer requested for the given uuid, None if not.
 
-        See ``add_arguments`` for the accepted options.
-
-        filter_kwargs is passed as an additional set of parameters that jobs can use to restrict
-        the database query used to retrieve relevant integrated channels.
+        Raises CommandError if uuid is invalid.
         """
-        # If a valid enterprise customer uuid was provided, transmit for that customer.
-        # Otherwise, transmit for all customers.
-        enterprise_customer = None
-        enterprise_customer_uuid = options['enterprise_customer']
-        if enterprise_customer_uuid:
+        if uuid is not None:
```

Comment:
It looks like you might be able to short-circuit this: 

```
if not uuid:
    return None
```

...then move on to the try/except